packages/@sanity/desk-tool/src/panes/documentPane/timeline/timelineItem.tsx

```diff
@@ -31,34 +47,42 @@ export function TimelineItem(props: {
   )
 
   return (
-    <li
-      className={styles.root}
+    <Root
+      data-ui="timelineItem"
+      radius={2}
       data-chunk-id={chunk.id}
-      data-state={state}
+      paddingY={0}
+      paddingX={2}
+      tone={isSelected || isWithinSelection ? 'default' : TIMELINE_ITEM_EVENT_TONE[type]}
```

Comment:
This is not taking into account that items should be blur when hovered.  So this condition should be something like:

```tsx
const [isHovered, setHovered] = useState(false)

<Root
  onMouseOver={() => setHovered(true)}
  onMouseOut={() => setHovered(false)}
  // ...
  tone={isHovered || isSelected || isWithinSelection ? 'default' : TIMELINE_ITEM_EVENT_TONE[type]}
  // ...
>

Reply:
For some reason the `onMouseOver` / `onMouseOut` wasn't working when on the root, but it was when it was pulled to a div that wrapped everything. So that's what I did :)